tests/test_patch_rebaser.py

```diff
@@ -322,3 +323,59 @@ def test_packages_to_process_skips_packages_not_in_the_list(
         main()
 
     mock_gitrepo.assert_not_called()
+
+
+def test_rebase_exception_gitreview(mock_repo):
+    mock_repo.branch.rebase_to_hash.side_effect = [
+        exceptions.RebaseException('.gitreview failed to rebase'),
+        True]
+    rebaser = Rebaser(mock_repo, "my_branch", "my_commit", "my_remote",
+                      "my_tstamp", dev_mode=True)
+
+    rebaser.try_automated_rebase_fix = Mock()
+    rebaser.try_automated_rebase_fix.side_effect = [True]
+    rebaser.perform_rebase()
+    rebaser.try_automated_rebase_fix.assert_called()
+
+
+def test_try_automated_rebase_fix(mock_repo):
+    rebaser = Rebaser(mock_repo, "my_branch", "my_commit", "my_remote",
+                      "my_tstamp", dev_mode=True)
+    rebaser._rebuild_gitreview = Mock()
+    exception = exceptions.RebaseException('.gitreview failed to rebase')
+    output = rebaser.try_automated_rebase_fix(exception)
+
+    mock_repo.git.rebase.assert_called()
+    rebaser._rebuild_gitreview.assert_called()
+    assert output == True
```

Comment:
Minor error in the CI on this, wants it to be:
```
assert output is True
```

Reply:
Oh, I didn't know it was required, actually it passed tests ocally without a complain :?. Anyway, it's fixed in the latest commit.